src/main/java/org/apache/fineract/api/OperationsDetailedApi.java

```diff
@@ -1,5 +1,6 @@
 package org.apache.fineract.api;
 
+import com.amazonaws.services.dynamodbv2.xspec.S;
```

Comment:
Please use an alternative dependency as AWS dependencies other than s3 will be removed.

Reply:
I think this change is not within the scope of this PR. Will take a look.

Reply:
Then this line should not come up in the code diff.